src/media/js/site/containers/login.js

```diff
@@ -4,9 +4,10 @@ import {bindActionCreators} from 'redux';
 
 import {fxaLoginBegin, login} from '../actions/login';
 import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
 
 
-export class Login extends React.Component {
+export class LoginForm extends React.Component {
```

Comment:
not really a fan. now half the codebase calls it Login, and the other half calls it LoginForm. and Login is implied to always be a form.